### ISSUE TITLE
Missing stats for dimensions

### ISSUE THREAD

**Issue by wonder-sk** [hobu/untwine#34]
Untwine currently does not output basic statistics for dimensions (Entwine does that by default). QGIS implementation is able to read those stats and use them to set up renderers (min/max values are especially useful to correctly set ranges for rendering):
```
      "count": 36802022,
      "minimum": 708379.5973094226,
      "maximum": 708446.8688094226,
      "mean": 708411.9621387576,
      "stddev": 19.869817066511605,
      "variance": 394.80963025663584
```

Additionally, for classification dimension (and maybe others?) Entwine also outputs `counts` - this would be also useful to allow auto-population of categories rather than requiring user to add them manually in the GUI:

```
      "counts": [
        {
          "count": 33137953,
          "value": 1
        },
        {
          "count": 3664069,
          "value": 2
        }
      ],
 ```

**Comment by wonder-sk** [comment]
The stats are actually not included in the official EPT format specification: https://entwine.io/entwine-point-tile.html

It is thought that EPT format could/should adopt support for optional extensions, and one of them would be support for stats as discussed here: https://github.com/connormanning/entwine/issues/149#issuecomment-447439609

**Comment by abellgithub** [comment]
Fixed with 7f4f6c976bf6b6319d29a25dc58b6343c9bba257
